packages/@adobe/spectrum-css-temp/components/button/index.css

```diff
@@ -167,7 +167,7 @@ governing permissions and limitations under the License.
 
     .spectrum-Button-label,
     .spectrum-Icon {
-      visibility: hidden;
+      opacity: 0;
```

Comment:
`visibility: hidden` hides the element from assistive technology so when the IDREF for the label or icon is used in `aria-labelledby` the accessibility name for the icon or label will be omitted from the accessibility name for the element referencing it. Using `opacity: 0` ensures that the elements will still render to the DOM and be available for reference using `aria-labelledby`.

Reply:
It kind of caught me by surprise too.